.travis.yml

```diff
@@ -138,6 +138,7 @@ matrix:
       name: "spellcheck"
       env: NAME=doxy-spellcheck
       install:
+        - sudo apt-get -y install libncursesw5
```

Comment:
Not crucial, but we can always install apt packages in a more declarative way: https://github.com/ARMmbed/mbed-os-example-blinky/blob/0016df36542bdcb2c84e4902b9c50364da8120a8/.travis.yml#L33-L41

Reply:
Added it via addons in the update